controllers/flowlogspipeline/flp_objects.go

```diff
@@ -245,46 +308,22 @@ func (b *builder) configMap() (*corev1.ConfigMap, string) {
 			{"name": "enrich",
 				"follows": "decode",
 			},
-			{"name": "encode",
-				"follows": "enrich",
-			},
 			{"name": "loki",
-				"follows": "encode",
+				"follows": "enrich",
 			},
-		},
-		"parameters": []map[string]interface{}{
-			ingest, decoder,
-			{"name": "enrich",
-				"transform": map[string]interface{}{
-					"type": "network",
-					"network": map[string]interface{}{
-						"rules": []map[string]interface{}{
-							{
-								"input":  "SrcAddr",
-								"output": "SrcK8S",
-								"type":   "add_kubernetes",
-							},
-							{
-								"input":  "DstAddr",
-								"output": "DstK8S",
-								"type":   "add_kubernetes",
-							},
-						},
-					},
-				},
+			{"name": "aggregate",
+				"follows": "enrich",
 			},
-			{"name": "encode",
-				"encode": map[string]interface{}{
-					"type": "none",
-				},
+			{"name": "prometheus",
+				"follows": "aggregate",
 			},
-			{"name": "loki",
-				"write": map[string]interface{}{
-					"type": "loki",
-					"loki": lokiWrite,
-				},
+			{"name": "write_none",
```

Comment:
what does "write_none" mean? Is it not going to expose the metrics to prom endpoint? or is that performed  via `prometheus` stage?

Reply:
There is an anomaly between the code that @mariomac wrote and the code that @KalmanMeth wrote that does not allow Prometheus to be finalizing stage. So this is empty "don't nothing" termination stage in the pipeline. I talked with @KalmanMeth on that and once they fix that we will be able to remove that empty stage.

Reply:
I added an issue in FLP repo